sdk/spring/azure-spring-boot-starter-active-directory/README.md

```diff
@@ -27,16 +27,21 @@ With Spring Starter for Azure Active Directory, now you can get started quickly
 [//]: # "{x-version-update-end}"
 
 ## Key concepts
-This package provides 2 ways to integrate with Spring Security and authenticate with Azure Active Directory.
+This package provides 2 ways to integrate with Spring Security and authenticate with Azure Active Directory, which are designed for scenarios of web application and resource server.
 
-The authorization flow is composed of 3 phrases:
+The authorization flow for web application includes:
+* Login with credentials by self-defined `azure` client registration and trigger **authorization code flow**. Application gets user's consent for all configured scopes except on-demand resources' scopes, and acquires an access token only for scopes of `openid, profile, offline_access, https://graph.microsoft.com/User.Read`.
+* When other resources are visited, associated clients will be loaded to trigger **refresh token flow** for authorization, and acquire an access token for configured scopes of that resource.
+* When on-demand resources are visited, associated clients will be loaded and trigger **authorization code flow** like the `azure` client.
+
+The authorization flow for resource server:
```

Comment:
Here and below introduction related with webapi will be udpated by @backwind1233

Reply:
@backwind1233 will we create a separate PR for this?

Reply:
@saragluna  I think so